fahadmak/django_iot_dashboard
# #167556574 Consume the thingspeak API

**What does this PR do?**
This PR ensures that the application can consume the thingspeak API.

**Description of Task to be completed?**
- create service.py to consume the API
- Add the channel feeds in dashboard view's context

**What are the relevant Pivotal Tracker stories?**
[#167556574](https://www.pivotaltracker.com/n/projects/2364284/stories/167556574)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd django_iot_dashboard`
- `git checkout ft-user-authentication-167556577`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `After login, you will be redirected to dashboard to view data from the API`

**Screenshots**
<img width="1021" alt="Room Availability - 600dp After" src="https://user-images.githubusercontent.com/13882936/62126410-4877c680-b2d8-11e9-961d-5277d669992e.png">
